modules/02-waydroid-modules.yml

```diff
@@ -3,8 +3,8 @@ type: shell
 commands:
 - cp -rT /sources/waydroid-modules/binder /usr/src/waydroid-binder-1
 - cp -rT /sources/waydroid-modules/ashmem /usr/src/waydroid-ashmem-1
-- dkms install waydroid-binder/1 -k 6.11.6-amd64
-- dkms install waydroid-ashmem/1 -k 6.11.6-amd64
+- dkms install waydroid-binder/1 -k $(ls -1 /usr/lib/modules | head -n 1)
```

Comment:
The changes in core image have been made.
This should now be `"$(< cat /usr/share/vanilla/kernel-version)"` I think